frontend/themes/hods/client-js/cookieSettings.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable max-len, no-var, vars-on-top, no-undef */
+'use strict';
+
+// TODO: update package.json(s)
```

Comment:
what do you need to update in package.json(s) would it be best to do as this PR?